JSON/Pixel Number Base.json

```diff
@@ -1,12 +1,12 @@
 {
-  "Author": "Panoptes (Xel), Crunchybot",
```

Comment:
Just checked the repo. It doesn't actually display it twice but it does show a little `?` when you hover, which isn't supposed to happen. If you could remove this line, that would be great.